packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss

```diff
@@ -162,7 +162,7 @@ $ms-DocumentCardActivity-personaTextGutter: 8px;
     top: -1px;
     right: -1px;
     bottom: -1px;
-    border: 2px solid #fafafa; // Match the background of the card
+    border: 2px solid $ms-color-white; // Match the background of the card
```

Comment:
do we need the border?

Reply:
I do not know.

Reply:
I think it would be wise to keep a visible border if there was one.